### PR TITLE
i18n support and improved visual rendering of the Web interface

### PR DESCRIPTION
**Pull request by gaetanlhf**
This PR adds i18n support and improves the visual rendering of the web interface.

## i18n
- Detection of the current browser language
- Translation via gettext
- Add the French translation

## Design
- Display of the number of available wikis
- Replacement of the default Bootstrap theme for the Zephyr theme from Bootswatch
- A small zoom effect on the cards when hovering
-  If a card is larger than another on a row, the others in that row now take the same size as this one

## Other
- Display of the current tag/version and the compilation date on the home page
- Cleaning

---

Demo (with the French translation):

https://user-images.githubusercontent.com/81755261/125353934-79395500-e363-11eb-9463-6e5ee1865109.mp4

### CODE REVIEW

**Comment by gaetanlhf** [comment]
What do you think about this PR? 🤔

File: html/static/templates/home.html.tpl [hunk 1]
```diff
@@ -21,12 +24,16 @@
...
+                            <a class="btn btn-outline-secondary mt-auto ms-auto" href="{{ .Link }}"><i class="fa fa-book fa-fw" aria-hidden="true"></i> {{ gettext "Read" }}</a>
                         </div>
                     </div>
                 </div>
             {{ end }}
 
     </div>
 
+    <div class="container position-absolute bottom-0 start-50 translate-middle-x py-2 d-flex justify-content-between bd-highlight mb-3" style="margin-bottom: 0 !important;">
+        <small class="text-muted">{{ .Version }} ({{ .BuildTime }})</small>
```
Comment (JojiiOfficial) [inline html/static/templates/home.html.tpl]: If built without proper `ldflags` this displays an empty `()` at the bottom. Could you check for them to be available and hide those parentheses if not available?

**Review by JojiiOfficial — COMMENTED** [review]
Rest looks fine. Thanks for the PR!
In my opinion a setting on the site should allow switching supported languages since some people might prefer a different language but this is just an idea.

File: html/static/templates/home.html.tpl (hunk 1 above)
Comment (gaetanlhf) [reply html/static/templates/home.html.tpl]: Oops, indeed I had forgotten to put some checks.
It's now corrected (see 04c6c1a). :)

**Comment by gaetanlhf** [comment]
> In my opinion a setting on the site should allow switching supported languages since some people might prefer a different language but this is just an idea.

Yes it's a good idea, a user might want to choose another language than the one of his browser (even if I think this case might be rare).
I'll keep this idea in mind for a future PR. :)

**Pull request merged**